openmc/mgxs/__init__.py

```diff
@@ -42,3 +44,409 @@
     3.6726e2, 9.069e2, 1.4251e3, 2.2395e3, 3.5191e3, 5.53e3,
     9.118e3, 1.503e4, 2.478e4, 4.085e4, 6.734e4, 1.11e5, 1.83e5,
     3.025e5, 5.e5, 8.21e5, 1.353e6, 2.231e6, 3.679e6, 6.0655e6, 2.e7])
+GROUP_STRUCTURES['VITAMIN-J-175'] = np.array([
```

Comment:
I see you've added a lower energy group to all these between 0 and the minimum group boundary of the actual set.  We should not include that as it will inevitably lead to a "one-off" error when a user is trying to count/identify groups or compare across references

Reply:
Some of these are defined in the reference as being generated via mathematical rules. I e., Equally spaced in a logarithm between x and y.  Would it be better to define with those rules where you can?

Reply:
> I see you've added a lower energy group to all these between 0 and the minimum group boundary of the actual set. We should not include that as it will inevitably lead to a "one-off" error when a user is trying to count/identify groups or compare across references

Thanks for taking a look. I included the 0 value to make it look similar to the CASMO group structures already in the file. The CASMO include a 0. value so I thought I should as well.

Reply:
I have removed the 0 value at the front of the new activation groups

Reply:
I have added references for two group structures where I could find a few more details on the mathematical rules

.. _CCFE-709 https://fispact.ukaea.uk/wiki/CCFE-709_group_structure
.. _UKAEA-1102 https://fispact.ukaea.uk/wiki/UKAEA-1102_group_structure